android/app/build.gradle

```diff
@@ -187,6 +187,8 @@ dependencies {
 
     implementation "androidx.swiperefreshlayout:swiperefreshlayout:1.0.0"
 
+    implementation project(':react-native-in-app-review')
+
```

Comment:
This should not be needed thanks to autolinking. Can you delete it and check that the feature still works?